sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/schema/JsonSchemaManager.java

```diff
@@ -0,0 +1,96 @@
+package io.stargate.sgv2.docsapi.service.schema;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.report.ProcessingMessage;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+import io.smallrye.mutiny.Uni;
+import io.stargate.bridge.proto.QueryOuterClass;
+import io.stargate.bridge.proto.StargateBridge;
+import io.stargate.sgv2.docsapi.api.common.StargateRequestInfo;
+import io.stargate.sgv2.docsapi.api.exception.ErrorCode;
+import io.stargate.sgv2.docsapi.api.exception.ErrorCodeRuntimeException;
+import io.stargate.sgv2.docsapi.service.schema.query.JsonSchemaQueryProvider;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+/**
+ * Json Schema manager provides basic operations to store, retrieve, and use the JSON schema for a
+ * Documents collection.
+ */
+@ApplicationScoped
+public class JsonSchemaManager {
+  @Inject ObjectMapper objectMapper;
+
+  @Inject JsonSchemaQueryProvider jsonSchemaQueryProvider;
+
+  @Inject StargateRequestInfo requestInfo;
+
+  private final JsonSchemaFactory jsonSchemaFactory = JsonSchemaFactory.byDefault();
+
+  public Uni<JsonNode> getJsonSchema(TableManager tableManager, String keyspace, String table) {
```

Comment:
I chose to use the TableManager and keyspace/table strings here, rather than a CqlTable, mostly to get the benefit of using authorization through the TableManager that is provided; also it might be more direct to just fetch the json schema out of the table directly rather than fetching the table and then fetching its comment...

But maybe just accessing the `comment` on a provided CqlTable works, too? Kind of makes a bunch of these methods very close to doing nothing though...